state/processor_test.go

```diff
@@ -383,7 +383,10 @@ func (s *ProcessorStateSuite) TestTransactionProcessor_Multilayer() {
 		createAccount(processor, toAddr(signers[1].PublicKey().Bytes()), 341578872634786, 10),
 		createAccount(processor, toAddr(signers[2].PublicKey().Bytes()), 1044987234, 0),
 	}
-
+	maxAmmount := map[types.Address]uint64{}
+	for _, acc := range accounts {
+		maxAmmount[acc.address] = acc.Balance() / uint64(testCycles*maxTransactions)
```

Comment:
i try to avoid division if possible. do you think it's possible to reverse the logic? i.e.randomly determine the amount of each transaction and make sure those accounts have enough balance before creating those accounts?

Reply:
good idea. i will just set the upper bound and for transferred amount and multiply it